packages/ComponentLibrary/src/components/Input/Select/index.tsx

```diff
@@ -0,0 +1,147 @@
+import React, { useState } from 'react';
+import {
+  Autocomplete,
+  AutocompleteRenderInputParams,
+  FormHelperText,
+  Paper,
+  PaperProps,
+  TextField,
+  Typography,
+} from '@mui/material';
+import { PRIMARY_CONTRAST, styles } from './style';
+import ChevronDown from '../../../assets/icons/chevron-down.svg';
+import XIcon from '../../../assets/icons/x.svg';
+import CheckIcon from '../../../assets/icons/check-circle-filled.svg';
+import { START_700 } from '../../../colors';
+import { NEUTRAL_90 } from '../../ConfigurationModal/style';
+import { ISelectInput } from './types';
+import './style.css';
+
+const CustomPaper: React.FC<PaperProps> = props => {
+  return <Paper {...props} sx={styles.optionsContainer} />;
+};
+
+const Select: React.FC<ISelectInput> = ({
+  title,
+  iconLeft,
+  options = [],
+  disabled = false,
+  helperText,
+  ...props
+}) => {
+  const [inputValue, setInputValue] = useState<string>('');
+  const [focused, setFocused] = useState<boolean>(false);
+
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setInputValue(event.target.value);
+  };
+
+  const handleFocus = () => {
+    setFocused(true);
+  };
+
+  const handleBlur = () => {
+    setFocused(false);
+    setInputValue('');
+  };
+
+  const handleSelectionChange = (value: any) => {
+    setInputValue(value?.title || '');
+  };
+
+  const getBackgroundFocus = (): string => {
+    if (!focused || inputValue) {
+      return 'transparent';
+    }
+    return PRIMARY_CONTRAST;
+  };
+
+  const renderInput = (params: AutocompleteRenderInputParams) => (
+    <TextField
+      {...params}
+      sx={{
```

Comment:
This styles shouldn't be in the style file declared as export 'const sx: { [key: string]: SxProps<Theme> } = {}' ?